hubspot/plugin.py

```diff
@@ -467,7 +471,10 @@ def _coerce_properties_to_lutra(
             if isinstance(value, datetime):
                 c_value = value
             elif isinstance(value, str):
-                c_value = datetime.fromisoformat(value)
+                if value != "":  # The value is an empty string when the date is not set
```

Comment:
i'd invert the check here

```
if value == "":
  ...
else:
 ...
```

so it's easier to follow and read from the comment